### ISSUE TITLE
[comments-module] Mention notifications result in UNKNOWN in path

### ISSUE THREAD

**Issue by emergingdzns** [pyrocms/pyrocms#4931]
**Describe the bug**
I've attached comments via the twig plugin to items in our module's feed of content. For example:
```
{% include "anomaly.module.comments::comments" with {'subject': item, 'approved':true} %}
```
The `item` is a model object. Comments are submitted properly and are displayed exactly correct. However, when a user is mentioned in the comment, the notification that goes out has a link that looks like this:
```
http://example.com/UNKNOWN#comment-38
```
The email does contain the subject and everything looks great. It seems as though the CommentRouter@view is failing to get the subject for some reason. I can't fix this because it's in repo. We just launched the app and suddenly the client discovered this.

**To Reproduce**
It's hard to give you steps because you'd have to install the app, create an account and then I would need to mention you. Instead I'll have to email you a copy of the email.

**Expected behavior**

**Screenshots**
![image](https://user-images.githubusercontent.com/1018133/53499613-8b99e900-3a5d-11e9-907d-0b2d7baa294c.png)
![image](https://user-images.githubusercontent.com/1018133/53499641-98b6d800-3a5d-11e9-9588-e3315b55e2a8.png)

**Comment by emergingdzns** [comment]
I'm wondering if I need to add something to my module's routes array that will tell the comment module how to setup the "view" route.

**Comment by RyanThompson** [comment]
Yes this needs to have a view route. You can do something like this within your addon: https://github.com/anomalylabs/posts-module/blob/2.6/src/PostsModuleServiceProvider.php#L97

Note the `as` portion is the name: https://github.com/anomalylabs/posts-module/blob/2.6/src/PostsModuleServiceProvider.php#L97

**Comment by emergingdzns** [comment]
Thanks @RyanThompson I am closer now to correcting it but I'm still a bit confused.

I added an `as` to my module so the array node looks like this:
```
        '/single/{id}/full' => [
            'as'   => 'gcmd.module.content::feed.view',
            'uses' => 'Gcmd\ContentModule\Http\Controller\NewsfeedController@getItemFull'
        ],
```
The model for the object that is being commented on is "feed" so I used "feed.view" after the module namespace. But I have a feeling that 'gcmd.module.content::feed.view' needs to be defined somewhere in order for this to work.

I am going through the posts module to try to follow the example but I can't figure out how that `as` string is translated such that the system knows to put the post slug in the {slug} segment of the url. In my case I'm using {id} which should be the id of the feed item that's being commented on.

Sorry for my lack of understanding.
